airflow-core/src/airflow/ui/src/i18n/config.ts

```diff
@@ -32,6 +32,7 @@ export const supportedLanguages = [
   { code: "nl", name: "Nederlands" },
   { code: "pl", name: "Polski" },
   { code: "zh-TW", name: "繁體中文" },
+  { code: "tr", flag: "🇹🇷", name: "Türkçe" },
```

Comment:
Flags are not used anymore
```suggestion
  { code: "tr", name: "Türkçe" },
```